integration_test/lit.cfg.py

```diff
@@ -206,6 +206,7 @@
   config.available_features.add('circt-lec')
   tools.append('circt-lec')
 
+config.substitutions.append(('%driver', f'{config.driver}'))
```

Comment:
Is this and the other lit change necessary given that this is hand-rolling a verilator call?

Reply:
We must expose the path to the driver somehow and it's in a subdirectory far away.